src/main/java/io/divolte/server/IncomingRequestProcessingPool.java

```diff
@@ -49,20 +45,20 @@
     private final Optional<KafkaFlushingPool> kafkaPool;
     private final Optional<HdfsFlushingPool> hdfsPool;
 
-    public IncomingRequestProcessingPool() {
-        this(ConfigFactory.load(), (e,b,r) -> {});
-    }
+//    public IncomingRequestProcessingPool() {
+//        this(ConfigFactory.load(), (e,b,r) -> {});
+//    }
```

Comment:
Hmm... It is apparently untouched and the build is green, so I think it's
out.

On Mon, Mar 9, 2015 at 1:56 PM, Andrew Snare <notifications@github.com>
wrote:

> In src/main/java/io/divolte/server/IncomingRequestProcessingPool.java
> <https://github.com/divolte/divolte-collector/pull/85#discussion_r26032804>
> :
>
> > @@ -49,20 +45,20 @@
> >      private final Optional<KafkaFlushingPool> kafkaPool;
> >      private final Optional<HdfsFlushingPool> hdfsPool;
> >
> > -    public IncomingRequestProcessingPool() {
> > -        this(ConfigFactory.load(), (e,b,r) -> {});
> > -    }
> > +//    public IncomingRequestProcessingPool() {
> > +//        this(ConfigFactory.load(), (e,b,r) -> {});
> > +//    }
>
> Is this code in or out? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/divolte/divolte-collector/pull/85/files#r26032804>.
>